_std/defines/gang.dm

```diff
@@ -0,0 +1,161 @@
+#define GANG_DRUG_SCORE_CATDRUGS 1
+#define GANG_DRUG_SCORE_METH 1.5
+
+
+/// what % of max HP a janktank revives people at
+#define JANKTANK2_DESIRED_HEALTH_PCT 0.15
+/// How long the JankTank must be prepared while using it on a gang member
+#define JANKTANK2_CHANNEL_TIME 5 SECONDS
+/// How long the janktank 2 sits in the corpse before revival - where it may be removed.
+#define JANKTANK2_PAUSE_TIME 5 SECONDS
+
+
+// GAMEMODE DEFINES
+#ifdef RP_MODE
+#define GANG_CRATE_INITIAL_DROP  30 MINUTES //!  when the first gang crate drops on RP
+#define GANG_CRATE_DROP_FREQUENCY 40 MINUTES //! how often gang crates are dropped on RP
+#else
+#define GANG_CRATE_INITIAL_DROP  25 MINUTES //!  when the first gang crate drops on classic
+#define GANG_CRATE_DROP_FREQUENCY 25 MINUTES //! how often gang crates are dropped on classic
+#endif
+
+
+#ifdef RP_MODE
+#define GANG_LOOT_INITIAL_DROP 10 MINUTES //! when the first gang duffel bag spawns on RP
+#define GANG_LOOT_DROP_FREQUENCY 15 MINUTES //! how often gang duffel bags are dropped on RP
+#else
+#define GANG_LOOT_INITIAL_DROP 7.5 MINUTES //! when the first gang duffel bag spawns on classic
+#define GANG_LOOT_DROP_FREQUENCY 15 MINUTES //! how often gang duffel bags are dropped on classic
+#endif
+#define GANG_LOOT_DROP_VOLUME_PER_GANG 2 //! how many duffel bags spawn, per gang
+
+
+
+// LAUNDERING DEFINES
+#define GANG_LAUNDER_DELAY 3 SECONDS //! how often gangs launder the money in their locker, in seconds
+#define GANG_LAUNDER_RATE 200 //! how much cash gets turned into points every elapsed GANG_LAUNDER_DELAY,
+#define GANG_LAUNDER_CAP 20000 //! how much cash can be in a locker at any given time?
+#define GANG_CASH_DIVISOR 5 //! How much cash is required for 1 gang point?
+
+
+
+// STREET CRED PURCHASE DEFINES:
+#define GANG_NEW_MEMBER_COST 500 		//! Cost of buying a new gang member from the locker
+#define GANG_NEW_MEMBER_COST_MULT 2.5 	//! How much buying a new gang member increases the price
+
+#define GANG_REVIVE_COST 750 		//! Cost of buying a revival syringe (JankTank II) from the locker
+#define GANG_REVIVE_COST_MULT 1.5 	//! How much buying a revival syringe increases its' price
+
+
+// CRATE DROP DEFINES
+#define GANG_CRATE_SCORE 500 //! how many points gang crates grant to each member, when opened
+#define GANG_CRATE_LOCK_TIME 300 SECONDS //! how long gang crates stay locked to the floor, in seconds
+
+#define GANG_LOOT_SCORE 300 //! how many points gang duffel bags grant to each member when opened
+
+
+// GANG TAG DEFINES:
+// Gang tags scan once every GANG_TAG_SCAN_RATE.
+// If they see a player, they will remember them until the next GANG_TAG_SCORE_INTERVAL.
+// Once the next GANG_TAG_SCORE_INTERVAL has elapsed, all memorised players provide heat, then are forgotten.
+
+/// how often gang tags search for nearby people, in deciseconds
+#define GANG_TAG_SCAN_RATE 10 DECI SECONDS
+/// how often tags calculate their heat & score, in seconds
+#define GANG_TAG_SCORE_INTERVAL 15 SECONDS
```

Comment:
```suggestion
/// how often gang tags search for nearby people
#define GANG_TAG_SCAN_RATE 1 SECOND
/// how often tags calculate their heat & score
#define GANG_TAG_SCORE_INTERVAL 15 SECONDS
```
comments are misleading since both values are in deciseconds